lib/daru/view/adapters/highcharts.rb

```diff
@@ -64,6 +64,16 @@ def export_html_file(plot, path='./plot.html')
           File.write(path, str)
         end
 
+        # @see #Daru::View::Plot.export_image
+        def export_image(plot, file_name='chart', type='pdf')
```

Comment:
Isn't we can find, if user running in IRuby notebook or console or web app?

Reply:
Yes, first I thought of creating only the `export_image` method. I tried to know whether the user is running in web app or IRuby notebook but couldn't do it. So, I created two different methods `export_image` and `export_image_iruby` instead